content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,277 @@
+✅  Service mlem-quick-start is up. You can check it out at https://mlem-quick-start.herokuapp.com/
+```
+
+# Fast model registry setup based on Git
+
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[mp]: /doc/start/metrics-parameters-plots
+[experiments]: /doc/user-guide/experiment-management
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[modeling process]: /doc/start/data-pipelines
+[remote storage]: /doc/command-reference/remote
+[sharing]: /doc/start/data-and-model-access
+[via cml]: https://cml.dev/doc/cml-with-dvc
+[gitops]: https://www.gitops.tech/
+
+MLEM is a core building block for a Git-based ML model registry, together with
+other Iterative tools, like GTO and DVC.
+
+ML model registries give your team key capabilities:
+
+- Collect and organize model [versions] from different sources effectively,
+  preserving their data provenance and lineage information.
+- Share metadata including [metrics and plots][mp] to help use and evaluate
+  models.
+- A standard interface to access all your ML artifacts, from early-stage
+  [experiments] to production-ready models.
+- Deploy specific models on different environments (dev, shadow, prod, etc.)
+  without touching the applications that consume them.
+- For security, control who can manage models, and audit their usage trails.
+
+Many of these benefits are built into DVC: Your [modeling process] and
+[performance data][mp] become **codified** in Git-based <abbr>DVC
+repositories</abbr>, making it possible to reproduce and manage models with
+standard Git workflows (along with code). Large model files are stored
+separately and efficiently, and can be pushed to [remote storage] -- a scalable
+access point for [sharing].
+
+To make a Git-native registry (on top of DVC or not), one option is to use [GTO]
+(Git Tag Ops). It tags ML model releases and promotions, and links them to
+artifacts in the repo using versioned annotations. This creates abstractions for
+your models, which lets you **manage their lifecycle** freely and directly from
+Git.
+
+```shell
+$ gto show
+╒══════════════════════╤══════════╤════════╤═════════╕
+│ name                 │ latest   │ #stage │ #prod   │
+╞══════════════════════╪══════════╪════════╪═════════╡
+│ pet-face-recognition │ v3.1.0   │ -      │ v3.0.0  │
+│ mlem-blep-classifier │ v0.4.1   │ v0.4.1 │ -       │
+│ dog-bark-translator  │ v0.0.1   │ -      │ v0.0.1  │
+╘══════════════════════╧══════════╧════════╧═════════╛
+
+$ mlem apply dog-bark-translator ./short-dog-phrase.wav
+🐶🚀🎉
+```
+
+# What next?
+
+Check out [MLEM on GitHub](https://github.com/iterative/mlem) today and let us
+know what you think! MLEM is a core building block for a Git-based model
+registry – for more information, visit our model registry page.
+
+Resources:
+
+- [Documentation](https://mlem.ai/doc)
+- [MLEM website](https://mlem.ai)
+- [MLEM on GitHub](https://github.com/iterative/mlem)
+- Building an ML model registry
```

Comment:
I guess we need to either
- forward to @jorgeorpinel's document in dvc docs about MR
- forward to https://iterative.ai/model-registry
WDYT?

Reply:
Link to https://iterative.ai/model-registry/ for now? Probably to Studio docs later

Reply:
Link to https://iterative.ai/model-registry/ for now? Probably to Studio docs later